README.md

```diff
@@ -12,11 +12,19 @@ https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-geocoder/
 
 ### Usage with a module bundler
 
-This module exports a single class called MapboxGeocoder as its default export,
-so in browserify or webpack, you can require it like:
+```bash
+npm install --save @mapbox/mapbox-gl-geocoder
```

Comment:
should we also mention `yarn add @mapbox/mapbox-gl-geocoder` to make it even easier for those using Yarn?

Reply:
I think those using yarn are familiar with npm. npm v7 brings workspaces too, which is the main reason for people choosing yarn over npm, AFAIK, so hopefully there will be more consolidation around npm afterwards.